### DISCUSSION TITLE
Check Point 為網路安全平台添加 AI、深度學習引擎，強化物聯網防禦

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#554] [General]
為強化物聯網安全防護，資安業者 Check Point 近日宣布推出全新網路安全平台 Check Point Quantum Titan，打造三款全新軟體刀鋒（software blade），利用人工智慧（AI）和深度學習技術打造進階威脅防禦，抵禦進階網域名稱系統（DNS）漏洞攻擊和網路釣魚，同時確保自動化物聯網安全。
同時，此一平台也是目前唯一能同時提供物聯網裝置探查（IoT device discovery），及自動部署零信任威脅防禦設定檔以防禦物聯網裝置的平台。
https://technews.tw/2022/10/28/tncap-toyota-corolla-cross/

**Comment by williamyen0623** [comment]
物聯網(IoT)目前普遍出現在每個家庭中，安全性的問題的確相當重要，感謝AI讓我們避免家電活過來攻擊我們？

**Comment by HsuChen88** [comment]
為了抵制AI過度的發展，難道這就是以AI制AI的策略嗎?
